backend/src/main/java/com/woowacourse/gongseek/article/application/TempArticleService.java

```diff
@@ -0,0 +1,100 @@
+package com.woowacourse.gongseek.article.application;
+
+import com.woowacourse.gongseek.article.domain.TempArticle;
+import com.woowacourse.gongseek.article.domain.repository.TempArticleRepository;
+import com.woowacourse.gongseek.article.exception.TempArticleNotFoundException;
+import com.woowacourse.gongseek.article.presentation.dto.TempArticleDetailResponse;
+import com.woowacourse.gongseek.article.presentation.dto.TempArticleIdResponse;
+import com.woowacourse.gongseek.article.presentation.dto.TempArticleRequest;
+import com.woowacourse.gongseek.article.presentation.dto.TempArticleResponse;
+import com.woowacourse.gongseek.article.presentation.dto.TempArticlesResponse;
+import com.woowacourse.gongseek.auth.exception.NotAuthorException;
+import com.woowacourse.gongseek.auth.exception.NotMemberException;
+import com.woowacourse.gongseek.auth.presentation.dto.AppMember;
+import com.woowacourse.gongseek.member.domain.Member;
+import com.woowacourse.gongseek.member.domain.repository.MemberRepository;
+import com.woowacourse.gongseek.member.exception.MemberNotFoundException;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@RequiredArgsConstructor
+@Service
+public class TempArticleService {
+
+    private final MemberRepository memberRepository;
+    private final TempArticleRepository tempArticleRepository;
+
+    @Transactional
+    public TempArticleIdResponse createOrUpdate(AppMember appMember, TempArticleRequest tempArticleRequest) {
+        validateGuest(appMember);
+        final Member member = getMember(appMember.getPayload());
```

Comment:
```suggestion
        final Member member = getMember(appMember);
```
AppMember 그대로 넘기는게 더 가독성이 좋을 것 같아요.

Reply:
getMember에서 getPayload()를 굳이 2번 호출할 필요가 없다고 생각해서 id값으로 넘겨줬어요!